config/swagger.yaml

```diff
@@ -2399,7 +2401,7 @@ definitions:
       name_hash:
         type: string
       recipient_pubkey:
-        type: string
+        $ref: '#/definitions/EncodedHashOrName'
```

Comment:
[Serialization doc mentions this as binary without size](https://github.com/aeternity/protocol/blob/0918c2a3262bb6e39e9eb589da380b248d006f3a/serializations.md#name-service-transfer-transaction) though the [AENS spec details size 32](https://github.com/aeternity/protocol/blob/0918c2a3262bb6e39e9eb589da380b248d006f3a/AENS.md#transfer). How does that reconcile with fitting a name?

Reply:
so, should we adjust spec, this is basically the feature this PR implements